bot/seasons/evergreen/recommend_game.py

```diff
@@ -0,0 +1,52 @@
+import json
+import logging
+import os
+from pathlib import Path
+from random import shuffle
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+DIR = "bot/resources/evergreen/game_recs"
+game_recs = []
+
+
+class RecommendGame(commands.Cog):
+    """Commands related to recommending games."""
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.populate_recs()
+
+    @commands.command(name="recommend_game")
+    async def recommend_game(self, ctx):
+        """Sends an Embed of a random game recommendation."""
+        if not game_recs:
+            self.populate_recs()
```

Comment:
It's inefficient to have to re-read the JSON files. Instead of popping items from the list, you can keep track of an index. Then, the list would be shuffled once the index reached the end.

Reply:
Not sure why I ignored using an index, it makes so much more sense.